src/app/app-design-tool/components/app-form-design/app-form-design.component.ts

```diff
@@ -0,0 +1,73 @@
+// Copyright 2018 Novo Nordisk Foundation Center for Biosustainability, DTU.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {AfterViewInit, Component, Input, OnInit, ViewChild} from '@angular/core';
+import {MatButton, MatSelect} from '@angular/material';
+import * as types from '../../../app-interactive-map/types';
+import {fromEvent, Observable, Subscription} from 'rxjs';
+import {select, Store} from '@ngrx/store';
+import {AppState} from '../../../store/app.reducers';
+import { NgForm } from '@angular/forms';
+import {FetchModelsDesign, FetchSpeciesDesign} from '../../store/design-tool.actions';
+import {activeModels} from '../../store/design-tool.selectors';
+
+
+@Component({
+  selector: 'app-form-design',
+  templateUrl: './app-form-design.component.html',
+  styleUrls: ['./app-form-design.component.scss'],
+})
+export class AppFormDesignComponent implements OnInit, AfterViewInit {
+  @ViewChild('designForm') designForm: NgForm;
+  subscription: Subscription;
+  @ViewChild('advanced') advancedButton: MatButton;
```

Comment:
You can use expansion panel https://material.angular.io/components/expansion/overview